src/module/keyboard-layouts/sv-latin.js

```diff
@@ -0,0 +1,20 @@
+//SPDX-FileCopyrightText: 2021 Martin (fohswe#8355), GH: plutoneld
+//
+// SPDX-License-Identifier: MIT
+
+export const svLatin = {
+  keycodeDisplayMapping: {
+    Backquote: '§',
+    Minus: '+',
+    IntlBackslash: '<',
+    Slash: '-',
+    Backslash: "'",
+    BracketLeft: 'Å',
+    Quote: 'Ä',
+    Semicolon: 'Ö',
+    Equal: '`',
+    BracketRight: '¨',
+  },
+
```

Comment:
maybe you want to add a binding mapping? For example, you could map `Minus` to `Slash`, so that bindings registered for `Minus` actually use the `-` key instead of the `+` key. I haven't checked all the cases, so there may be more things you want to map.

Reply:
tbh i didn't quite get the binding mapping at first, but i think. I think that is the only one making sense to map though.